test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ContactDetailsConfirmAddressControllerSpec.scala

```diff
@@ -137,6 +151,21 @@ class ContactDetailsConfirmAddressControllerSpec extends ControllerSpec {
         status(result) mustBe OK
         businessRegisteredAddressPopulatedSameAs(incorporationDetails.companyAddress)
       }
+
+      "throw internal server exception when organisation type does not match" in {
```

Comment:
Can we get a similar test in for Partnerships too please?

Reply:
Probably will have to wait for the Ryan's PR to be merged to write this test